SingularityUI/app/components/status/StatusPage.jsx

```diff
@@ -121,58 +120,46 @@ class StatusPage extends React.Component {
   }
 
   getRequestsData(model) {
-    return model.requests.map((r) => {
+    return model.requests.map((request) => {
       return (
         {
-          component: Link,
-          beforeFill: r.type,
-          prop: {
-            text: `${r.count} ${r.label} ${this.renderPercentage(r.count, model.totalRequests)}`,
-            url: r.link,
-            value: r.count,
-            id: r.type
-          }
+          component: (className) => (
+            <a href={request.link} className={className}>
+              {request.count} {request.label} {this.renderPercentage(request.count, model.totalRequests)}
+            </a>
+          ),
+          beforeFill: request.type,
+          value: request.count,
+          id: request.type
         }
       );
     });
   }
 
   getTasksData(model) {
-    const res = model.tasks.map((t) => {
+    const res = model.tasks.map((task) => {
       return (
       {
-        component: Link,
-        beforeFill: t.type,
-        prop: {
-          text: `${t.count} ${t.label} ${this.renderPercentage(t.count, model.totalTasks)}`,
-          url: t.link,
-          value: t.count,
-          id: t.type
-        }
+        component: (className) => (
+          <a href={task.link} className={className}>
```

Comment:
I'm trying to get rid of the `atomicDisplayItems` components - that was on purpose.